supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJobDefinition.scala

```diff
@@ -32,13 +32,18 @@
 package cromwell.backend.impl.aws
 
 import scala.language.postfixOps
+import scala.collection.mutable.ListBuffer
 import cromwell.backend.BackendJobDescriptor
-import cromwell.core.path.Path
+import cromwell.backend.io.JobPaths
 import software.amazon.awssdk.services.batch.model.{ContainerProperties, KeyValuePair}
 import wdl4s.parser.MemoryUnit
 
 import scala.collection.JavaConverters._
 
+// GZip
```

Comment:
TOL not really necessary to comment this, when we optimize the imports with IntelliJ the meaning will be lost anyway 🙂